synthetics/data_source_agents.go

```diff
@@ -68,3 +94,38 @@ func agentsToMaps(agents []synthetics.V202101beta1Agent) []interface{} {
 	}
 	return result
 }
+
+func filterAgents(agents []synthetics.V202101beta1Agent, d *schema.ResourceData) []synthetics.V202101beta1Agent {
+	lat, latExists := d.GetOk(latitudeKey)
+	lon, lonExists := d.GetOk(longitudeKey)
+	minDist, minDistExists := d.GetOk(minDistanceKey)
+	maxDist, maxDistExists := d.GetOk(maxDistanceKey)
+
+	if !minDistExists {
+		minDist = MinDistDefault
```

Comment:
You can have the 0.0 as a direct constant here too. I cannot think of a  situation in which we'd need a different default minimal distance. Can you?

Reply:
Hmm, not really, good point. Addressed: 7aa5e0c5df172373543f675d9fbb0d93e71f7eaf